### ISSUE TITLE
crash | fabric 1.20.1

### ISSUE THREAD

**Issue by Surpos0** [mariumbacchus/Soulslike-Weaponry#71]
im sure there is conflict with one of my mods but i don't know witch one cuz, before 1.0.0 version mod was working fine so idk
But heres log:
[log.txt](https://github.com/mariumbacchus/Soulslike-Weaponry/files/13369529/log.txt)

**Comment by ThunderWolfUE** [comment]
**Majrusz's Enchantments** causes the newest update to crash. **Better Nether** and **Goblin Traders** might be incompatible too? Try deleting these mods, then if it's working fine you can try reinstalling better nether and goblin traders

**Comment by mariumbacchus** [comment]
Are you sure this is caused by Soulslike Weaponry? It doesn't mention it's name once in the crash log and it references onScheduledTick method which is used in fluids and chests... which this mod doesn't add.

**Comment by Surpos0** [comment]
yes you were right Majrusz's Enchantments was causeing issue. thx for help
